structure.Rmd

```diff
@@ -355,20 +355,22 @@ In both cases we see two active libraries, consulted in this order:
 1.  A user library
 2.  A system-level or global library
 
-This setup is typical on Windows, but is something you usually need to opt into on macOS [^structure-3].
+This setup is typical on Windows, but is something you usually need to opt into on macOS and Linux[^structure-3].
 With this setup, add-on packages installed from CRAN (or elsewhere) or under local development are kept in the user library.
 Above, the macOS system is used as a primary development machine and has many packages here (\~1000), whereas the Windows system is only used occasionally and is much more spartan.
-The core set of base and recommended packages that ship with R live in the system-level library and are the same on macOS and Windows.
+The core set of base and recommended packages that ship with R live in the system-level library and are the same on all operating systems.
 This separation appeals to many developers and makes it easy to, for example, clean out your add-on packages without disturbing your base R installation.
 
-[^structure-3]: For more details, see the [Maintaining R section](https://whattheyforgot.org/maintaining-r.html#how-to-transfer-your-library-when-updating-r) in *What They Forgot To Teach You About R*.
+[^structure-3]: For more details, see the [Maintaining R section](https://whattheyforgot.org/maintaining-r.html#how-to-transfer-your-library-when-updating-r) in *What They Forgot To Teach You About R*, [Managing Libraries](https://cran.r-project.org/doc/manuals/r-release/R-admin.html#Managing-libraries) in *R Installation and Administration* and the R help files for `Startup` and `.libPaths()`.
```

Comment:
I am actively mulling over what to do about links into WRE: link to CRAN's version or our Quarto-ified version (<https://rstudio.github.io/r-manuals/r-exts/>, which I think is much more pleasant to read)? This is being tracked in #888 and we certainly aren't going to decide that here. This is just an FYI.

I think these tweaks are more consistent with what we're doing elsewhere.

```suggestion
[^structure-3]: For more details, see the [Maintaining R section](https://whattheyforgot.org/maintaining-r.html#how-to-transfer-your-library-when-updating-r) in *What They Forgot To Teach You About R*, [Managing Libraries](https://cran.r-project.org/doc/manuals/r-release/R-admin.html#Managing-libraries) in *R Installation and Administration* and the R help files for `?Startup` and `?.libPaths`.
```

Reply:
Yes, the quarto version of WRE is much nicer to read - though this link is not to WRE rather R administration.

Reply:
Ah, I see. We Quarto-ized all the manuals, so it'll be the same decision:

https://rstudio.github.io/r-manuals/r-admin/

Reply:
Oh I didn't realize that - lovely!